lib/sfrest/site_ownership.rb

```diff
@@ -37,5 +37,19 @@ def make_default_owner(username)
     def remove_default_owner
       @conn.delete('/api/v1/site-ownership')
     end
+
+    # Transfers site ownership.
+    # @param [Integer] site id.
+    # @param [String] username.
+    # @param [String] email.
```

Comment:
Is this parameter missing from function definition?

Reply:
We need only one parameter out of two, either username or email.

Reply:
Then we should specify that in the description and add the necessary logic to send the `email` parameter as well. Currently this only works for the username usecase. There's no way to use it using the email usecase.
Also, we should extend the rspec to test both the username and email usecases.

Reply:
Fixed :)